src/app/components/auth/login/login.component.html

```diff
@@ -1,12 +1,61 @@
-<div class="login-container">
-  <app-input [isRequired]="true" [label]="'Username'" [type]="'text'" [theme]="'dark'" [icon]="'assets/images/username_dark.png'"  #formlogin></app-input>
-  <app-input [isRequired]="true" [label]="'Password'" [type]="'password'" [theme]="'dark'" [icon]="'assets/images/password_dark.png'"  #formlogin></app-input>
-  <div class="auth-forgot-div">
-    <span class="btn btn-fire" (click)="formValidate('formlogin')">LOGIN</span>
-    <span class="auth-forgot">Forgot password or username?</span>
-  </div>
-  <div class="auth-signup-row">
-    <span>Start with a new account </span>
-    <a [routerLink]="['/auth/signup']" >Sign Up</a>
-  </div>
+<div class="col s12 m6 offset-m3 left">
+  <ul class="left">
+    <li class="ev-horiz-list"><a routerLink="/auth/login" class="dark-link" routerLinkActive="active-auth"
```

Comment:
Get `/auth/login` etc. (also check for other such instances in the file) from constant variables in the `.ts` file.